packages/schema/src/atoms/index.ts

```diff
@@ -77,3 +78,7 @@ export const ReviewIdSchema = z.number().int().positive();
 export const RestaurantAddressSchema = z.string().min(1).max(255);
 
 export const RestaurantDescriptionSchema = z.string().min(1).max(255);
+
+const UrlSchema = z.string().url();
+
+export const RestaurantUrlSchema = UrlSchema;
```

Comment:
I think this export should match the style of the rest of the file.
```suggestion
export const RestaurantUrlSchema = z.string().url();
```

Reply:
Separated UrlSchema out here to reflect that RestaurantUrlSchema is just a regular URL schema - the other schemas are more specific, hence why I didn't extract anything out from them.